lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Player::Human < Player
+  def move(board)
+    puts "#{self.token}, please enter your move as a number 1-9."
+    input = gets.strip
+    input
+  end
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board)
  puts "#{self.token}, please enter your move as a number 1-9."
  gets.strip
end
```